api/naming_service.go

```diff
@@ -29,6 +30,8 @@ func MakeLookupRoute(router gin.IRouter) {
 	ns.GET("/lookup", handleLookup)
 
 	TLDMapping[".eth"] = CoinType.ETH
+	TLDMapping[".xyz"] = CoinType.ETH
```

Comment:
@hewigovens might need to drop `.` inside the mapping.

Does this mean anyone can create top level domains?

Reply:
no, those two are supported by ens now, it can support more tlds